<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1545</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>隐写加密</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="encodeButton">
              <property name="text">
               <string>加密</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QGraphicsView" name="graphicsView">
              <property name="minimumSize">
               <size>
                <width>700</width>
                <height>700</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="downButton">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>下载加密图片</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="chooseButton">
                <property name="text">
                 <string>选择图片</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="methodBox">
                <item>
                 <property name="text">
                  <string>LSB</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QGraphicsView" name="preView">
                <property name="minimumSize">
                 <size>
                  <width>700</width>
                  <height>700</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="messageLine"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>隐写解密</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="3">
           <layout class="QVBoxLayout" name="verticalLayout_5"/>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="chooseButton_2">
                <property name="text">
                 <string>选择图片</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="methodBox_2">
                <item>
                 <property name="text">
                  <string>LSB</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QGraphicsView" name="graphicsView_2">
                <property name="minimumSize">
                 <size>
                  <width>700</width>
                  <height>700</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="decodeButton">
              <property name="text">
               <string>解密</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <widget class="QGraphicsView" name="outputView_2">
            <property name="minimumSize">
             <size>
              <width>700</width>
              <height>700</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
